openy_gated_content.post_update.php

```diff
@@ -126,3 +126,45 @@ function openy_gated_content_post_update_eventseries_livestream(&$sandbox) {
 function openy_gated_content_post_update_eventseries_meeting(&$sandbox) {
   _openy_gated_content_permissions($sandbox, 'eventseries', 'virtual_meeting', 'id');
 }
+
+/**
+ * Set gated_content paragraphs title and description if it was empty.
+ */
+function openy_gated_content_post_update_paragraph_headline(&$sandbox) {
+  $prgf_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $gated_content_prgf = $prgf_storage->loadByProperties(['type' => 'gated_content']);
+  $gated_content_prgf = end($gated_content_prgf);
+  if ($gated_content_prgf) {
+    $page_id = $gated_content_prgf->parent_id->value;
+
+    $header_prgf = $prgf_storage->loadByProperties([
```

Comment:
same here, lot of content = memory issues. We should load by chunks ( 50 )

Reply:
@podarok we will find one `gated_content` paragraph, get `parent_id `and load the only one banner related to this page with `gated_content`. This banner will be deleted because we have a new one in the application.

cc @retif